list_of_subprojects.md

```diff
@@ -0,0 +1,70 @@
+# List of Official Jupyter Subprojects
+
+At the highest level, any Github repository under any official Jupyter GitHub organization is either itself an Official Jupyter Subproject, or is part of an Official Jupyter Subproject. All such Subprojects have all of the privileges and responsibilities detailed in the Software Subprojects of Jupyter governance document. Within the Official Jupyter Subproject designation, there are two types of Subprojects: ones with a formal decision making body and SSC representation, and smaller, less active ones whose decision making body is the SSC itself. This document enumerates Subprojects of these two types.
+
+### Official Subprojects with SSC representation
+
+The following Jupyter Subprojects have their own formal decision making body that elects and maintains an SSC representative:
+
+- JupyterLab (https://github.com/jupyterlab/jupyterlab)
+- JupyterHub and Binder 
+  - https://github.com/jupyterhub/jupyterhub
+  - https://github.com/jupyterhub/binder
+  - https://github.com/jupyterhub/binderhub
+- Voilà (https://github.com/voila-dashboards/voila)
+- Jupyter Server (https://github.com/jupyter-server/jupyter_server)
+- Jupyter Widgets (https://github.com/jupyter-widgets)
+- Jupyter Notebook (https://github.com/jupyter/notebook)
```

Comment:
I don't think Notebook should be listed, given that it's in maintenance mode.

Reply:
Typing for @afshin and @fperez in the Friday meeting: the notebook is significant enough from our user's perspective that it probably belongs here for now, but as its usage and development ramps down, we can reconsider that.

Reply:
Setting status to resolved since Zach +1'd the response.